packages/flutter/test/foundation/assertions_test.dart

```diff
@@ -59,21 +59,6 @@ void main() {
         '═════════════════════════════════════════════════════════════════\n',
 
     );
-    expect(
```

Comment:
rather than just reducing the number of tests, let's replace them with tests that check similar codepaths, e.g. adding `exception: NullError()` to these, or pass in some other object with a known toString, or something.